modulesets/gstreamer.modules

```diff
@@ -97,6 +97,15 @@
     <branch repo="slomo" module="gst-player" checkoutdir="gst-player"/>
   </autotools>
 
+  <autotools id="gst-editing-services" autogenargs="--disable-gtk-doc">
+    <dependencies>
+      <dep package="libxml2"/>
+      <dep package="gstreamer"/>
+      <dep package="gst-devtools"/>
```

Comment:
1. Is gst-devtools really needed for this?
2. Doesn't gst-editing-services require gst-plugins-*?